dev/devicelab/bin/tasks/hello_world__start_up.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
nit: 2019

Reply:
It was copied from another file, so the year in the copyright stays the same.  Filed under: things I learned from @Hixie 🙂